docs/view_components/energy_system_component.rst

```diff
@@ -8,17 +8,16 @@ Attributes
 .. Please refer to the definition of what an attribute is in the tool_interface.rst fileg
 .. The properties should be filled in only if applicable.
 
-:ref:`energy-type-def`
-
-    Energy type linked to the different energy sectors (e.g. heat, electricity, ...)
+**Energy type**
+    ref:`energy-type-def` linked to the different energy sectors (e.g. heat, electricity, ...)
 
 
 **Component type**
-    Can be one of (Source, sink, transformer, storage, bus)
+    Can be one of [Source, Sink, Transformer, Storage, Bus, ...] .. we propably not going to only use oemof.solph basic components in the tool, so list needs to be extended
```

Comment:
I agree, however the ... implies for me that somewhere else the full list of components exists and I wonder then where is that list. I the list is closed it is clear that these are all values a component can take up to now (this list can however become bigger in the future). We can describe the component types more in detail somewhere else in RTD and link to this place after the open-ended list

Reply:
I agree, with this point. I opened issue #80 to continue this discussion.